community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -513,4 +513,9 @@ private static String defaultPageCacheMemory()
     @Internal
     public static final Setting<Integer> batch_inserter_batch_size = setting( "batch_inserter_batch_size", INTEGER,
             "10000" );
+
+    @Description( "Create an archive of an index before re-creating it if failing to load on startup." )
+    @Internal
+    public static final Setting<Boolean> archive_failed_index = setting(
+            "unsupported.dbms.index.archive_failed", BOOLEAN, "true" );
```

Comment:
@thobe Not to archive them. That takes away the possible disaster scenario of (e.g.) out of file handles turning into out of disk. But it allows us to ask an operator to turn it on to help diagnose a case where indexes repeatedly fail.